rfc/11/README.md

```diff
@@ -19,13 +19,17 @@ Interact with Motr cluster.
 Commands:
 
     bootstrap   bootstrap the cluster
+    drive-state post drive event to Motr
+    fetch-fids  Fetch the fids for services configured on a node
```

Comment:
Would be nice to add which fids we are fetching motr, s3 etc

Reply:
Done